doc/sphinx-guides/source/api/native-api.rst

```diff
@@ -2992,9 +2994,62 @@ Deletes an ``AuthenticatedUser`` whose ``id``  is passed. ::
 
     DELETE http://$SERVER/api/admin/authenticatedUsers/id/$id
     
-Note: If the user has performed certain actions such as creating or contributing to a Dataset or downloading a file they cannot be deleted.
+Note: If the user has performed certain actions such as creating or contributing to a Dataset or downloading a file they cannot be deleted. If a user cannot be deleted for this reason, you can choose to :ref:`disable a user<disable-a-user>`.
+    
+.. _disable-a-user:
+
+Disable a User
+~~~~~~~~~~~~~~
+
+Disables an ``AuthenticatedUser`` whose ``identifier`` (without the ``@`` sign) is passed. ::
+
+    PUT http://$SERVER/api/admin/authenticatedUsers/disable/$identifier
     
+Disables an ``AuthenticatedUser`` whose ``id``  is passed. ::
+
+    PUT http://$SERVER/api/admin/authenticatedUsers/disable/id/$id
     
+Note: Since the primary purpose of Dataverse is to serve as an archive and to track data accesses and the modifications of data and metadata, a simple mechanism to delete users that have performed certain actions in the system is not provided by design. 
+
+Disabling a user with this endpoint will:
+
+- Disable the user's ability to log in to the Dataverse installation
+- Remove the user's access from all datasets and files
+- Cancel any pending file access requests generated by the user
+- Remove the user from all groups
+- No longer have notifications generated or sent by the Dataverse installation
```

Comment:
and additional things:

- Remove all roles (permissions) for the user.
- ...
- ...
- ...

(sorry, out of time)